kge/cli.py

```diff
@@ -272,9 +279,10 @@ def main():
             dataset = Dataset.load(config)
 
             # let's go
-            job = Job.create(config, dataset)
             if args.command == "resume":
-                job.resume(checkpoint_file)
+                job = Job.resume(checkpoint_file, config=config)
+            else:
+                job = Job.create(config, dataset)
```

Comment:
If it's optional, we do not use to compute it here at all

Reply:
fixed